CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # 6.7.0 (unreleased)
 
 ## Breaking changes
+* **Slack**: Remove Mention setting and instead introduce _Mention Users_, _Mention Groups_, and _Mention Channel_. The first two settings require user and group IDs respectively. This change was necessary due to the fact that the way of mentioning via the Slack API changed, and mentions in Slack notifications no longer worked.
```

Comment:
You could add a link mentioning the modification in the Slack API.

Reply:
@papagian Done.